setup.py

```diff
@@ -25,7 +25,7 @@ def get_version():
     url="https://github.com/executablebooks/sphinx-tabs",
     license="MIT",
     python_requires="~=3.6",
-    install_requires=["sphinx>=2,<4", "pygments"],
+    install_requires=["sphinx>=2,<5", "pygments", "docutils<=0.16.0"],
```

Comment:
Did you mean to say the following?

```suggestion
    install_requires=["sphinx>=2,<5", "pygments", "docutils~=0.16.0"],
```

Reply:
Wasn't sure if we wanted to allow lower versions of docutils or not. Probably fine.